scripts/open-coredump.sh

```diff
@@ -377,7 +377,7 @@ Launching dbuild container.
 
 To examine the coredump with gdb:
 
-    $ gdb -x scylla-gdb.py -ex 'set directories /src/scylla' -iex 'set solib-search-path /opt/scylladb/libreloc/' --core ${COREFILE} /opt/scylladb/libexec/scylla
+    $ gdb -x scylla-gdb.py -ex 'set directories /src/scylla' -iex 'set solib-search-path /opt/scylladb/libreloc/' -iex 'set substitute-path /jenkins/workspace/scylla-${MAIN_BRANCH}/next/scylla /workdir' --core ${COREFILE} /opt/scylladb/libexec/scylla
```

Comment:
This will be incorrect for production coredumps, on the other hand if the substitution rule doesn't match, GDB will simply ignore it, so it won't hurt.

Note that `set directories` uses `/src/scylla`, maybe both should use the same path.

BTW won't this make the line be wrapped on the terminal? Currently, I can select the entire hint and just copy to my terminal on my screen.

Reply:
> This will be incorrect for production coredumps, 

indeed. i took a closer look at the workspace directories used by the promotion jobs and the release jobs, and realized that they are actually using different naming conventions.

> on the other hand if the substitution rule doesn't match, GDB will simply ignore it, so it won't hurt.
> 
> Note that `set directories` uses `/src/scylla`, maybe both should use the same path.

sure. more consistent this way.

> 
> BTW won't this make the line be wrapped on the terminal? Currently, I can select the entire hint and just copy to my terminal on my screen.

i wrapped the lines for better user experience.